pyomo/core/tests/unit/test_var_set_bounds.py

```diff
@@ -152,6 +151,8 @@ def Xtest_list_domain(self):
         self.model.con2 = Constraint(expr=self.model.y[2] <= 2.9)
         
         self.instance = self.model.create_instance()
+        # !! THIS SEEMS LIKE A BUG!! - mrmundt
+        # I think it's supposed to be SolverFactory
         self.opt = solver["glpk"]
```

Comment:
I think you're right but it's extra weird that this test isn't failing. Do we have `glpk` in our testing environment? Could you check if this test is being run or not?

Reply:
It's not running. But I also tried it locally - nosetests doesn't pick up any of the names because they all start with `X`. So someone turned off all these tests.

Reply:
DECISION: We're going to handle all `Xtest` instances in a separate PR.